cypress/e2e/events.cy.ts

```diff
@@ -1,97 +1,306 @@
 import * as dayjs from "dayjs";
+import 'dayjs/locale/da'
 import "cypress-if";
 
-const events = {
-  singleEvent: {
-    title: "Single event",
-    subtitle: "A subtitle",
-    recurType: "Custom/Single Event",
-    start: dayjs("2030-01-01T10:00:00"),
-    end: dayjs("2030-01-01T16:00:00"),
+dayjs.locale('da')
+
+// The fields of the eventseries entity, that we use when creating and comparing.
+export interface EventType {
+  uuid?: string,
+  title?: string,
+  subtitle: string,
+  recurType: 'custom' | 'weekly_recurring_date',
+  ticketManagerRelevance: boolean,
+  state: string,
```

Comment:
Consider if this should be a set of allowed strings like you provided for `recurType`.